server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -425,7 +428,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
         }
     }
 
-    public close(closeType: LambdaCloseType) {
+    public async close(closeType: LambdaCloseType) {
```

Comment:
Is it necessary to mark this as async? Since there's no await's in this method, I don't think it has to be.
Also [the code](https://github.com/microsoft/FluidFramework/blob/6b89ebfb7be8ccfcdbc81e9f45a2429b1fc0f997/server/routerlicious/packages/lambdas-driver/src/document-router/documentPartition.ts#L122) that is calling this close method is not async and is not awaiting this method so it might have some undesired consequences.

Reply:
Got it. Thanks for sharing! We removed this async. :)